### PR TITLE
#1677 Character Report Part 2 - Show all character issues from other exercises

### PR DESCRIPTION
**Pull request by HalcyonJAC**
## What's included?
On Character Issues page show all character issues from other exercises.

https://docs.google.com/document/d/1KumfgugHW0PQWasyjjs1X07Eh4uMbyPXqHeo_gLu3gk/edit

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Go to character issues report of an exercise
2. Check character issues from other exercises

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://user-images.githubusercontent.com/79906532/180025742-77a2abcd-43f7-4810-a823-1d57a2d82766.mov

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

**Review by andrew-isaac — COMMENTED** [review]
Fantastic work ! This can be moved to UT lane :)

**Review by warrensearle — CHANGES_REQUESTED** [review]
Looks like there are some lint fixes needed plus please see comment regarding using constants and camelCase for our database values

**Comment by tomlovesgithub** [comment]
Functions great, but agree with warrens points re: constants and db convention! Will re review after those have been addressed.

File: src/views/Exercise/Reports/CharacterIssues.vue [hunk 1]
```diff
@@ -213,31 +214,108 @@
...
+                    Multiple professional conducts
+                  </option>
+                  <option value="Single other matter">
+                    Single other matter
+                  </option>
+                  <option value="Multiple other matters">
+                    Multiple other matters
+                  </option>
+                  <option value="Mixed">
+                    Mixed
```
[the remark this answers is not in the document]
Comment (HalcyonJAC) [reply src/views/Exercise/Reports/CharacterIssues.vue]: Thank you for your feedback.

**Review by warrensearle — APPROVED** [review]
Hey @HalcyonJAC this is much better, thank you 👍 

Approved, though we will need to keep an eye on this one. The `getOtherApplicationRecords` method loops through all the applications and for each one queries firebase to obtain all the candidate's other applications. Should be ok but there's now quite a lot of document reads going on in the front end.

For other reports we went with the pattern of generating a single `report` document containing all the data needed. I can see how this approach may not be quite right for the functionality here (storing outcomes and notes) however is something to keep in mind as this functionality evolves.

**Review by tomlovesgithub — APPROVED** [review]

**Review by andrew-isaac — APPROVED** [review]

**Pull request merged**